raycicmd/util.go

```diff
@@ -13,6 +14,15 @@ func boolInMap(m map[string]any, key string) (bool, bool) {
 	return b, ok
 }
 
+func stringHasPrefix(prefixes []string, key string) bool {
```

Comment:
nit: maybe put `key` first? as "string `key` has prefix `prefixes`? and rename `key` to `s`, as it is just a string, not a key?

the other ones here, like `stringInMap`, the string actually means "type" of the value. like `stringInMap` is checking if `m[key]` exists, and it is a string. quite a bit different from the one here.